docs/cluster_manifest.md

```diff
@@ -0,0 +1,215 @@
+  examples](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+  for details on tolerations and possible values of those keys. When set, this
+  value overrides the `pod_toleration` setting from the operator. Optional.
+
+### Postgres parameters
+
+Those parameters are grouped under the `postgresql` top-level key.
+
+* **version**
+  the postgres major version of the cluster. Looks at the [Spilo
+  project](https://github.com/zalando/spilo) for the list of supported
+  versions. Changing the cluster version once the cluster has been bootstrapped
+  is not supported. Required field.
+
+* **parameters**
+  a dictionary of postgres parameter names and values to apply to the resulting
+  cluster. Optional (Spilo automatically sets reasonable defaults for
+  parameters like work_mem or max_connections).
+
+### Patroni parameters
+
+Those parameters are grouped under the `patroni` top-level key. See the [patroni
+documentation](https://patroni.readthedocs.io/en/latest/SETTINGS.html) for the
+explanation of `ttl` and `loop_wait` parameters.
+
+* **initdb**
+  a map of key-value pairs describing initdb parameters. For `data-checksum`,
+  `debug`, `no-locale`, `noclean`, `nosync` and `sync-only` parameters use
+  `true` as the value if you want to set them. Changes to this option do not
+  affect the already initialized clusters. Optional. 
+
+* **pg_hba**
+  list of custom `pg_hba` lines to replace default ones. Note that the default
+  ones include
+
+  ```
+  hostsll all +pamrole all pam
+  ```
+  , where pamrole is the name of the role for the pam authentication; any
+    custom `pg_hba` should include the pam line to avoid breaking pam
+    authentication. Optional.
+
+* **ttl**
+  patroni `ttl` parameter value, optional. The default is set by the Spilo
+  docker image. Optional.
+
+* **loop_wait**
+  patroni `loop_wait` parameter value, optional. The default is set by the
+  Spilo docker image. Optional.
+
+* **retry_timeout**
+  patroni `retry_timeout` parameter value, optional. The default is set by the
+  Spilo docker image. Optional.
+
+* **maximum_lag_on_failover**
+  patroni `maximum_lag_on_failover` parameter value, optional. The default is
+  set by the Spilo docker image. Optional.
+
+### Postgres container resources
+
+Those parameters define CPU and memory requests and limits for the postgres
+container. They are grouped under the `resources` top-level key. There are two
+subgroups, `requests` and `limits`.
+
+#### Requests
+
+CPU and memory requests for the postgres container.
+
+* **cpu**
+  CPU requests for the postgres container. Optional, overrides the
+  `default_cpu_requests` operator configuration parameter. Optional.
+
+* **memory**
+  memory requests for the postgres container. Optional, overrides the
+  `default_memory_request` operator configuration parameter. Optional.
+
+#### Limits
+
+CPU and memory limits for the postgres container.
```

Comment:
imo worth a [link to k8s docs](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/), just to clarify what type of request is meant here